docs/design/autodiff.md

```diff
@@ -260,7 +260,7 @@ intermediaries which don't have concrete variables. When we add a pair (A, C) an
 
  ```C
 
-void f_SSA_Rev(inout DP<float> dpa, inout DP<float> dpb, float dout)
+void f_SSA_Rev(inout DP<float> dpa, inout DP<float> dpb, float doubt)
```

Comment:
I think this is `dout` correct

Reply:
I didn't mean to include that change, thanks for catching it. Fixed.